tests/e2e/paper/paper-view-model.ts

```diff
@@ -0,0 +1,106 @@
+import { type Locator, type Page } from "@playwright/test";
+
+export class PaperViewPageModel {
+    readonly page: Page;
+    readonly informationButton: Locator;
+    readonly documentButton: Locator;
+    readonly showMoreInformationButton: Locator;
+    readonly showLessInformationButton: Locator;
+    readonly allListEntries: Locator;
+    readonly generalInformationLabel: Locator;
+    readonly publicationTypeLabel: Locator;
+    readonly publicationNameLabel: Locator;
+    readonly externalIdLabel: Locator;
+
+    readonly projectName: string;
+    readonly paperNames: string[];
+
+    constructor(page: Page) {
+        this.page = page;
+        this.informationButton = page.getByRole("tab", { name: "Information", exact: true });
+        this.documentButton = page.getByRole("tab", { name: "Document", exact: true });
+        this.showMoreInformationButton = page.getByRole("button", {
+            name: "Show more information",
+            exact: true,
+        });
+        this.showLessInformationButton = page.getByRole("button", {
+            name: "Show less information",
+            exact: true,
+        });
+        this.allListEntries = page.getByRole("listitem");
+        this.generalInformationLabel = page.getByText("General Information");
+        this.publicationTypeLabel = page.getByText("Publication Type");
+        this.publicationNameLabel = page.getByText("Publication Name");
+        this.externalIdLabel = page.getByText("External ID");
+
+        this.projectName = "Project 1";
+        this.paperNames = ["Paper View Paper 0", "Paper View Paper 1"];
```

Comment:
```suggestion
        this.paperNames = ["Paper View - Paper 0", "Paper View - Paper 1"];
```

This makes it easier to understand, and it doesn't sound weird.